src/test/java/teammates/sqlui/webapi/GetFeedbackQuestionRecipientsActionTest.java

```diff
@@ -0,0 +1,257 @@
+package teammates.sqlui.webapi;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.when;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.FeedbackParticipantType;
+import teammates.common.datatransfer.FeedbackQuestionRecipient;
+import teammates.common.util.Const;
+import teammates.storage.sqlentity.FeedbackQuestion;
+import teammates.storage.sqlentity.FeedbackSession;
+import teammates.storage.sqlentity.Instructor;
+import teammates.storage.sqlentity.Student;
+import teammates.ui.output.FeedbackQuestionRecipientsData;
+import teammates.ui.request.Intent;
+import teammates.ui.webapi.EntityNotFoundException;
+import teammates.ui.webapi.GetFeedbackQuestionRecipientsAction;
+import teammates.ui.webapi.InvalidHttpParameterException;
+import teammates.ui.webapi.JsonResult;
+
+/**
+ * SUT: {@link GetFeedbackQuestionRecipientsAction}.
+ */
+public class GetFeedbackQuestionRecipientsActionTest extends BaseActionTest<GetFeedbackQuestionRecipientsAction> {
+
+    private FeedbackSession typicalFeedbackSession;
+    private FeedbackQuestion typicalFeedbackQuestion;
+    private Student typicalStudent;
+    private Instructor typicalInstructor;
+    private Map<String, FeedbackQuestionRecipient> typicalRecipients;
+
+    @Override
+    protected String getActionUri() {
+        return Const.ResourceURIs.QUESTION_RECIPIENTS;
+    }
+
+    @Override
+    protected String getRequestMethod() {
+        return GET;
+    }
+
+    @BeforeMethod
+    void setUp() {
+        typicalFeedbackSession = getTypicalFeedbackSessionForCourse(getTypicalCourse());
+        typicalFeedbackQuestion = getTypicalFeedbackQuestionForSession(typicalFeedbackSession);
+        typicalStudent = getTypicalStudent();
```

Comment:
```suggestion
        typicalStudent = getTypicalStudent();
        typicalStudent.setAccount(getTypicalAccount());
```

If there is no account, getGoogleId() always returns null